lib/repositories/match_details_repository.dart

```diff
@@ -324,4 +325,53 @@ class MatchDetailsRepository {
     ];
     return response.map(LineUpModel.fromJson).toList();
   }
+
+  Future<List<StatisticsModel>> fetchMatchStats(String matchID) async {
+    // return (await liveMatchesRemoteService.fetchMatchStats(matchID: matchID)).response;
```

Comment:
Commented code

Reply:
left it to not overuse Api limited usage 😅